plugins/inputs/logfile/tail/tail.go

```diff
@@ -593,7 +596,7 @@ func (tail *Tail) exitOnDeletion() {
 // with the last chunk of variable size.
 func partitionString(s string, chunkSize int) []string {
 	if chunkSize <= 0 {
-		panic("invalid chunkSize")
+		panic("Invalid chunkSize")
```

Comment:
It does not write log before panicking. Same criteria that I mentioned before: `most encountered errors has been at surface levels instead of internal package and avoid having unnecessary logs`. However, if it is needed, I would add writing to logs for this as well

Reply:
Why do we not want to write any log for this if it panics? Are you saying that telegraf is catching this before terminating?